packages/db/prisma/schema.prisma

```diff
@@ -134,6 +134,7 @@ model AdminUser {
   role          Role?
   accounts      AdminAccount[]
   sessions      AdminSession[]
+  audit         Audit[]
```

Comment:
should be `audits` or `audit_list/logs` to differentiate from a single row (1-to-1 rel for instance)

Reply:
Better audit_logs